incubator/bootstrap_cli/__main__.py

```diff
@@ -635,33 +766,35 @@ def create_group(
             Group: the new created CDF Group
         """
 
-        aad_source_id, aad_source_name = None, None
-        if aad_mapping:
+        idp_source_id, idp_source_name = None, None
+        if idp_mapping:
             # explicit given
             # TODO: change from tuple to dataclass
-            assert len(aad_mapping) == 2
-            aad_source_id, aad_source_name = aad_mapping
+            assert len(idp_mapping) == 2
+            idp_source_id, idp_source_name = idp_mapping
         else:
             # check lookup from provided config
-            aad_source_id, aad_source_name = self.aad_mapping_lookup.get(group_name, [None, None])
-
-        # print(f"=====  group_name<{group_name}> | aad_source_id<{aad_source_id}>
-        # | aad_source_name<{aad_source_name}> ===")
+            mapping = self.bootstrap_config.get_idp_cdf_mapping_for_group(
+                cdf_project=self.cdf_project, cdf_group=group_name
+            )
+            # unpack
+            idp_source_id, idp_source_name = mapping.idp_source_id, mapping.idp_source_name
 
         # check if group already exists, if yes it will be deleted after a new one is created
         old_group_ids = self.get_group_ids_by_name(group_name)
 
         new_group = Group(name=group_name, capabilities=group_capabilities)
-        if aad_source_id:
+        if idp_source_id:
             # inject (both will be pushed through the API call!)
-            new_group.source_id = aad_source_id  # 'S-314159-1234'
-            new_group.source = aad_source_name  # type: ignore # 'AD Group FooBar' # type: ignore
+            new_group.source_id = idp_source_id  # 'S-314159-1234'
+            new_group.source = idp_source_name  # type: ignore
 
         # print(f"group_create_object:<{group_create_object}>")
         # overwrite new_group as it now contains id too
         if self.is_dry_run:
             _logger.info(f"Dry run - Creating group with name: <{new_group.name}>")
-            _logger.debug(f"Dry run - Creating group: <{new_group}>")
+            _logger.info(f"Dry run - Creating group details: <{new_group}>")
```

Comment:
I woud prefer having this on Debug level, to have a possibility to get a more readable overview when filtering on info